index.js

```diff
@@ -185,6 +185,7 @@ class TestingFramework {
 
             const isValidResponse = step.bot.some(expectedResponses => {
                 if (expectedResponses.length !== response.length) {
+                    console.error(`RESPONSES LENGHT DOES NOT MATCH:\nEXPECT: ${expectedResponses.length}\nACTUAL: ${response.length}`);
```

Comment:
Fix grammar & typo:

RESPONSES LENGTHS DO NOT MATCH

Reply:
fixed to 
RESPONSES LENGTH DOES NOT MATCH